src/holdem/parse.rs

```diff
@@ -522,6 +523,33 @@ impl RangeParser {
 
         Ok(filtered)
     }
+
+    /// Parse a string and return all the starting hands
+    ///
+    /// # Examples
+    ///
+    /// Same as `parse_one` but this will parse a comma separated list of hands.
+    ///
+    /// ```
+    /// use rs_poker::holdem::RangeParser;
+    ///
+    /// let hand = RangeParser::parse("KK+,A2s+").unwrap();
+    /// assert_eq!(60, hand.len());
+    ///
+    /// // Filters out duplicates.
+    /// assert_eq!(RangeParser::parse_one("AK-87s,A2s+").unwrap().len(), 72)
+    /// ```
+    pub fn parse(r_str: &str) -> Result<Vec<Hand>, RSPokerError> {
```

Comment:
What about `parse_all` or `parse_any`? The other naming is explicit about the number of hands to expect so we should continue that pattern.

Reply:
Maybe `parse_many`?

Reply:
Sounds good to me.